This patch adds an #include def in order for net-tools 1.60 to properly compile

--- linux-2.4.32/include/linux/if_fddi.h.avx	2006-01-11 02:16:23.000000000 -0700
+++ linux-2.4.32/include/linux/if_fddi.h	2006-01-11 02:16:23.000000000 -0700
@@ -24,6 +24,8 @@
 #ifndef _LINUX_IF_FDDI_H
 #define _LINUX_IF_FDDI_H
 
+#include <linux/netdevice.h>
+
 /*
  *  Define max and min legal sizes.  The frame sizes do not include
  *  4 byte FCS/CRC (frame check sequence).
